src/nvim/api/win_config.c

```diff
@@ -603,6 +692,25 @@ static bool parse_float_config(Dict(float_config) *config, FloatConfig *fconfig,
     return false;
   }
 
+  if (HAS_KEY(config->title_pos)) {
+    if (!HAS_KEY(config->title)) {
+      api_set_error(err, kErrorTypeException, "title_pos must set with title");
```

Comment:
a little bit grammar. maybe "`'title_pos' requires 'title' to be set"`

Reply:
fixed.